addons/mail/models/models.py

```diff
@@ -99,7 +107,7 @@ def _format_notification(self, subject, message, doc=False):
     @api.model
     def _get_admin_notification(self, failure_type, module_messages=None):
         """
-            To simplify failure notification process, this method will work as wrapper and will 
```

Comment:
Maybe all this "rewritting without html"  could already be done in first commit, where you add the _get_admin_notification method (and some others) so we avoid rewritting the same code in the same PR.

Reply:
same thing as (*)!